payments-core/src/main/java/com/stripe/android/view/StripeEditText.kt

```diff
@@ -33,7 +33,8 @@ open class StripeEditText @JvmOverloads constructor(
     attrs: AttributeSet? = null,
     defStyleAttr: Int = androidx.appcompat.R.attr.editTextStyle
 ) : TextInputEditText(context, attrs, defStyleAttr) {
-    protected var isLastKeyDelete: Boolean = false
+    @VisibleForTesting(otherwise = VisibleForTesting.PROTECTED)
+    internal var isLastKeyDelete: Boolean = false
```

Comment:
Could you reduce the visibility for the setter with this?

```
    @VisibleForTesting(otherwise = VisibleForTesting.PROTECTED)
    internal var isLastKeyDelete: Boolean = false
        protected set
```

Reply:
This should already be covered by the `VisibleForTesting.PROTECTED`, no?

Reply:
It actually won’t let me set the `protected set` visibility, unless I remove the `internal` visibility first 😕

Reply:
Good to know, you can ignore this then 😄